msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -758,14 +774,85 @@ The `publisherDocsUrl` is a HTTPS URL to an information page for admins to get g
 
 Specifies the SaaS offer associated with your app.
 
-|Name| Type| Maximum size | Required | Description|
+|Name| Type|Maximum size|Required|Description|
 |---|---|---|---|---|
 |`offerId`| string | 2,048 characters | ✔ | A unique identifier that includes your Publisher ID and Offer ID, which you can find in [Partner Center](https://partner.microsoft.com/dashboard). You must format the string as `publisherId.offerId`.|
 
+## meetingExtensionDefinition
+
+**Optional** - object
+
+Specify meeting extension definition. For more information, see [custom Together Mode scenes in Teams](../../apps-in-teams-meetings/teams-together-mode.md).
+
+|Name| Type| Maximum size | Required | Description|
+|---|---|---|---|---|
+|`scenes`|array of objects| 5 items||Meeting supported scenes.|
+
+### meetingExtensionDefinition.scenes
+
+|Name| Type|Maximum size|Required |Description|
+|---|---|---|---|---|
+|`id`|||✔| The unique identifier for the scene. This id must be a GUID. |
+|`name`| string | 128 characters |✔| The name of the scene. |
+|`file`|||✔| The relative file path to the scenes' metadata json file. |
+|`preview`|||✔| The relative file path to the scenes' PNG preview icon. |
+|`maxAudience`| integer | 50  |✔| The maximum number of audiences supported in the scene. |
+|`seatsReservedForOrganizersOrPresenters`| integer | 50 |✔| The number of seats reserved for organizers or presenters.|
+
+## authorization
+
+**Optional** — object
+
+Specify and consolidate authorization related information for the app.
+
+|Name| Type|Maximum size|Required |Description|
+|---|---|---|---|---|
+|`permissions`||||List of permissions that the app needs to function.|
+
+### authorization.permissions
+
+|Name| Type|Maximum size|Required |Description|
+|---|---|---|---|---|
+|`resourceSpecific`| array of objects|16 items||Permissions that guard data access on resource instance level.|
+
+### authorization.permissions.resourceSpecific
+
+|Name| Type|Maximum size|Required |Description|
+|---|---|---|---|---|
+|`type`|string||✔| The type of the resource-specific permission. Options: `Application` and `Delegated`.|
+|`name`|string|128 characters|✔|The name of the resource-specific permission. <br> For more information, see [Application permissions](../../graph-api/rsc/resource-specific-consent.md) and [Delegated permissions](#delegated-permissions).|
+
+### Delegated permissions
+
+* **Resource-specific permissions for teams**
+
+    |**Delegated permission**|**Action**|
+    |---|---|
+    |`ChannelMeetingParticipant.Read.Group`| Read the participants of the teams' channel meetings.|
+    |`InAppPurchase.Allow.Group`| Show and complete in-app purchases for users in the team.|
+    |`ChannelMeetingStage.Write.Group`| Show content on the meeting stage of channel meetings associated with the team.|
+
+* **Resource-specific permissions for chats or meetings**
+
+    |**Delegated permission**|**Action**|
```

Comment:
Description

Reply:
For the permission descriptions, please lift and shift from the following table:

<html>
<body>
<!--StartFragment--><div ccp_infra_version='3' ccp_infra_timestamp='1643891171411' ccp_infra_user_hash='3504234856' ccp_infra_copy_id='566c28cc-e345-45df-8f7d-8d9219216403' data-ccp-timestamp='1643891171411'><html>

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<style>
table
	{mso-displayed-decimal-separator:"\.";
	mso-displayed-thousand-separator:"\,";}
tr
	{mso-height-source:auto;}
col
	{mso-width-source:auto;}
td
	{padding-top:1px;
	padding-right:1px;
	padding-left:1px;
	mso-ignore:padding;
	color:black;
	font-size:11.0pt;
	font-weight:400;
	font-style:normal;
	text-decoration:none;
	font-family:Calibri, sans-serif;
	mso-font-charset:0;
	text-align:general;
	vertical-align:bottom;
	border:none;
	white-space:nowrap;
	mso-rotate:0;}
.xl39
	{white-space:normal;}
.xl40
	{color:black;
	white-space:normal;}
.xl142
	{background:yellow;
	mso-pattern:black none;
	white-space:normal;}
</style>
</head>

<body link="#0563C1" vlink="#954F72">



OnlineMeetingParticipant.ToggleIncomingAudio.Chat | Toggle incoming audio for participants in meetings associated with this chat | Allows the app to toggle incoming audio for participants in meetings associated with this chat, on behalf of the signed-in user.
-- | -- | --
CameraStream.Read.User | Read the user's camera stream | Allows the app to read the user's camera stream, on behalf of the signed-in user.
OutgoingVideoStream.Write.User | Modify the user's outgoing video | Allows the app to modify the user's outgoing video, on behalf of the signed-in user.
InAppPurchase.Allow.User | Show and complete in-app purchases | Allow the app to show the user marketplace offers and complete the user's purchases within the app, on behalf of the signed-in user.
MeetingStage.Write.Chat | Show content on the meeting stage of meetings associated with this chat | Allows the app to show content on the meeting stage in meetings associated with this chat, on behalf of the signed-in user.
OnlineMeetingParticipant.Read.Chat | Read participant information, including name, role, id, joined and left times, of meetings associated with this chat | Allows the app to read participant information, including name, role, id, joined and left times, of meeting associated with this chat, on behalf of the signed-in user.
ChannelMeetingParticipant.Read.Group | Read participant information, including name, role, id, joined and left times, of channel meetings associated with this team | Allows the app to read participant information, including name, role, id, joined and left times, of channel meetings associated with this team, on behalf of the signed-in user.
ChannelMeetingStage.Write.Group | Show content on the meeting stage of channel meetings associated with this team | Allows the app to show content on the meeting stage in channel meetings associated with this team, on behalf of the signed-in user.
InAppPurchase.Allow.Group | Show and complete in-app purchases for users in this team | Allow the app to show marketplace offers to users in this team and complete their purchases within the app, on behalf of the signed-in user.
InAppPurchase.Allow.Chat | Show and complete in-app purchases for users in this chat and any associated meeting | Allow the app to show marketplace offers to the users in this chat, and any associated meeting, and complete their purchases within the app, on behalf of the signed-in user.



</body>

</html>
</div><!--EndFragment-->
</body>
</html>